extensions/wikia/RobotsTxt/classes/WikiaRobots.class.php

```diff
@@ -62,10 +62,7 @@ class WikiaRobots {
 	 * @var int[]
 	 */
 	private $blockedNamespaces = [
```

Comment:
the code below adds `$wgRobotsTxtCustomRules['disallowNamespace']` to blockedNamespaces. should we still handle blocked namespaces in robots.txt code?